cmd/bandd/cmd/root.go

```diff
@@ -84,27 +82,19 @@ func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
 func initRootCmd(rootCmd *cobra.Command, encodingConfig params.EncodingConfig) {
 	rootCmd.AddCommand(
 		InitCmd(band.NewDefaultGenesisState(), band.DefaultNodeHome),
-		genutilcli.CollectGenTxsCmd(banktypes.GenesisBalancesIterator{}, band.DefaultNodeHome),
-		genutilcli.GenTxCmd(
-			band.ModuleBasics,
-			encodingConfig.TxConfig,
-			banktypes.GenesisBalancesIterator{},
-			band.DefaultNodeHome,
+		// 0.47 TODO: check validity of this function
+		genesisCommand(
+			encodingConfig,
+			AddGenesisDataSourceCmd(band.DefaultNodeHome),
+			AddGenesisOracleScriptCmd(band.DefaultNodeHome),
 		),
-		genutilcli.ValidateGenesisCmd(band.ModuleBasics),
-		AddGenesisAccountCmd(band.DefaultNodeHome),
-		AddGenesisDataSourceCmd(band.DefaultNodeHome),
-		AddGenesisOracleScriptCmd(band.DefaultNodeHome),
 		tmcli.NewCompletionCmd(rootCmd, true),
 		// testnetCmd(band.ModuleBasics, banktypes.GenesisBalancesIterator{}),
 		debug.Cmd(),
 		config.Cmd(),
```

Comment:
please add pruning and snapshot command as well. e.g.

```
		pruning.Cmd(newApp, simapp.DefaultNodeHome),
		snapshot.Cmd(newApp),
```

Reply:
ok